lessons/11-basic-data-structs/notes/hashes.md

```diff
@@ -0,0 +1,95 @@
+# Hashes
+
+A `Hash` is another very popular collection type in Ruby, similar to that Array.
+It is similar to an Array, except that indexing is done via unordered keys of any object type, not an ordered integer index.
+
+```ruby
+{first_num: 1, greeting: "Hello", pi: 3.14}
+```
```

Comment:
I think something here needs to be added to explain what a key-value pairing is and how this can be seen in the example you provided here. Explained in English words, not code.